nala/studio/acom/full-pricing-express/full-pricing-express.page.js

```diff
@@ -0,0 +1,186 @@
+export default class ACOMFullPricingExpressPage {
+    constructor(page) {
+        this.page = page;
+
+        // Collection selectors
+        this.collection = page.locator('merch-card-collection, .card-collection, .pricing-collection');
+        this.collectionTitle = page.locator('.collection-title');
+
+        // Card element selectors for full-pricing-express variant only
+        // Use specific selectors that match the actual DOM structure
+        this.cardTitle = page.locator('merch-card h3, merch-card h2, merch-card h4, merch-card h1');
+        this.cardIcon = page.locator('merch-card[variant="full-pricing-express"] mas-mnemonic').first();
+        this.cardPrice = page.locator('merch-card span[is="inline-price"]:not([data-template="strikethrough"])');
+        this.cardPriceStrikethrough = page.locator('[slot="price"] span.price-strikethrough');
+        this.cardStrikethroughPrice = page.locator(
+            '[slot="price"] span[is="inline-price"][data-template="strikethrough"], [slot="price"] span.price-strikethrough',
+        );
+        this.cardPriceLegal = page.locator('span[is="inline-price"][data-template="legal"]');
+        this.cardBody = page.locator('merch-card p');
+        this.cardDescription2 = page.locator('merch-card[variant="full-pricing-express"] div[slot="description2"]').first();
```

Comment:
could you pls remove description2 from the model and instead use short description and description?